### DISCUSSION TITLE
Moving entire side of LEDs

### DISCUSSION THREAD

**Discussion by tungmeister** [awawa-dev/HyperHDR#712] [Q&A]
Hi, in my layout I need to move the entire right side of LEDs up by a small amount as they are slightly out of alignment - how would I go about this? The Top, bottom and left are all perfectly aligned it's just how the strip has ended up being physically positioned along the right edge meaning the position of that entire side needs shifting up by about 15mm.
![image](https://github.com/awawa-dev/HyperHDR/assets/127809/4fe21765-25fa-4856-b419-36a90b886601)

**Comment by awawa-dev** [comment]
Hi
First, export a backup of your configuration.
Right-click on one LED on the right, then select properties and note the "left" and "right" values.
![obraz](https://github.com/awawa-dev/HyperHDR/assets/69086569/5d5f0f3e-ef5a-451f-81ea-fe855e9c0d3d)
On the same page, you have the "Generated/Current LED Configuration" tab, copy its contents and paste into a text editor of your choice that has an advanced text replacement function (e.g. Visual Studio Code)
Replace
```
    "hmax": 1,
    "hmin": 0.95,
```

with eg.

```
    "hmax": 0.97,
    "hmin": 0.92,
```
both text must be treated as one whole block. Paste again the result in HyperHDR into the same tab and save it.
The final result after saving & refresh:
![obraz](https://github.com/awawa-dev/HyperHDR/assets/69086569/31749d4d-65d9-4362-9df9-17e69f6c3f27)
